comet/broker.py

```diff
@@ -23,6 +23,7 @@
 DEFAULT_PORT = 12050
 
 app = Sanic(__name__)
+logger.propagate = False
```

Comment:
Why do we want Sanic logging to not propagate? Will it have its own handler?

Reply:
Yeah, without this line, I get the lines from the sanic logger twice. Once in their format and once in the one configured in the scripts.